web/source/text/processor.ts

```diff
@@ -495,6 +495,7 @@ namespace com.keyman.text {
         if(keyman.isEmbedded) {
           // A special embedded callback used to setup direct callbacks to app-native code.
           keyman['oninserttext'](ruleTransform.deleteLeft, ruleTransform.insert, ruleTransform.deleteRight);
+          keyman.refreshElementContent(outputTarget.getElement());
```

Comment:
I note that this here doesn't test for presence of `refreshElementContent`?

Reply:
Yup.  It's only implemented in `kmwEmbedded.ts`.

Reply:
It's been a part of KMW a long, long time - since before the `keyman.isEmbedded` flag existed.